### ISSUE TITLE
p5 install fails on python3.10 due to PyOpenGL-Accelerate==3.1.5 dependency

### ISSUE THREAD

**Issue by khimaros** [p5py/p5#395]
**Describe the bug**
`python3 -m pip install p5` fails on python3.10 (debian testing)

**To Reproduce**
`python3 -m pip install p5`

**Expected behavior**
install should succeed

**Screenshots**
```
      src/vbo.c: In function ‘__pyx_tp_dealloc_17OpenGL_accelerate_3vbo_VBOOffset’:
      src/vbo.c:11619:5: error: lvalue required as increment operand
      11619 |     ++Py_REFCNT(o);
            |     ^~
[...]
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for PyOpenGL-accelerate
```

**System information:**
 - p5 release (version number or latest commit): 0.8.0
 - Python version: 3.10
 - Operating system: debian testing

**Additional context**
`python3 -m pip install PyOpenGL-Accelerate==3.1.6` works

blocked on: https://github.com/mcfletch/pyopengl/issues/78

workaround is to install from git commit `git+https://github.com/mcfletch/pyopengl.git@227f9c66976d9f5dadf62b9a97e6beaec84831ca#subdirectory=accelerate`

**Comment by khimaros** [comment]
workaround to install p5:

```
python3 -m pip install --no-deps 'freetype-py==2.1.0.post1' 'glfw==1.11.2' 'numpy>=1.18.5' 'Pillow==9.0.1' 'vispy==0.10.0' 'git+https://github.com/mcfletch/pyopengl.git@227f9c66976d9f5dadf62b9a97e6beaec84831ca#subdirectory=accelerate' 'PyOpenGL==3.1.6' 'requests>=2.25.0' 'dataclasses;python_version=="3.6"' 'skia-python>=85.0' p5
```

**Comment by tushar5526** [comment]
Hi, thanks for submitting the issue. I think we have to refactor our dependencies. You can also raise a PR to fix this one if you want.

**Comment by tushar5526** [comment]
Closed by #398
